Makefile

```diff
@@ -64,6 +64,7 @@ unit-test-nocoverage-host: golint-host govet-host
 
 build: golint govet
 	vagrant ssh mon0 -c 'sudo -i sh -c "cd $(GUESTGOPATH); make run-build"'
+	make run
```

Comment:
Yes; I thought it made more sense there, since after a build you will want to restart it anyways, for manual testing purposes.